diff -Naur linux-4.16.1-orig/include/linux/mmc/card.h linux-4.16.1/include/linux/mmc/card.h
--- linux-4.16.1-orig/include/linux/mmc/card.h	2018-04-08 21:29:52.000000000 +0900
+++ linux-4.16.1/include/linux/mmc/card.h	2018-04-10 15:57:29.969909765 +0900
@@ -151,7 +151,9 @@
 	unsigned int		hs_max_dtr;
 	unsigned int		uhs_max_dtr;
 #define HIGH_SPEED_MAX_DTR	50000000
-#define UHS_SDR104_MAX_DTR	208000000
+//#define UHS_SDR104_MAX_DTR	208000000
+// Downclock UHS. Blame SWMobile for faulty HW
+#define UHS_SDR104_MAX_DTR	150000000
 #define UHS_SDR50_MAX_DTR	100000000
 #define UHS_DDR50_MAX_DTR	50000000
 #define UHS_SDR25_MAX_DTR	UHS_DDR50_MAX_DTR
